nuxeo-runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/model/impl/ExtensionPointImpl.java

```diff
@@ -102,7 +103,13 @@ public Extension createExtension(Element element) {
                     }
                 }
             }
-            contribs = xmap.loadAll(new XMapContext(extension.getContext()), extension.getElement());
+            try {
+                contribs = xmap.loadAll(new XMapContext(extension.getContext()), extension.getElement());
+            } catch (XMapException e) {
+                throw new RuntimeException(
+                        e.getMessage() + " while processing component: " + extension.getComponent().getName().getName(),
+                        e);
```

Comment:
For readability, I'd rather output in the order "Kind of failure, involved component, error message". Also, I'd prefer to make use of `ComponentName.toString()`.

    throw new RuntimeException(String.format("Failed to process component %s:\n\t%s",
                        extension.getComponent().getName(), e.getMessage()));